aptos-move/e2e-move-tests/tests/rotate_auth_key.rs

```diff
@@ -31,48 +31,82 @@ struct RotationProofChallenge {
 }
 
 #[test]
-fn rotate_auth_key() {
+fn rotate_auth_key_twice() {
     let mut harness = MoveHarness::new();
+    let mut account1 = harness.new_account_with_key_pair();
 
-    let account1 = harness.new_account_at(AccountAddress::from_hex_literal("0x123").unwrap());
-    let address = account1.address();
-    let new_private_key = Ed25519PrivateKey::generate_for_testing();
-    let new_public_key = new_private_key.public_key();
-    let new_auth_key = AuthenticationKey::ed25519(&new_public_key);
+    let account2 = harness.new_account_with_key_pair();
+    // assert that the payload is successfully processed (the signatures are correct)
+    assert_successful_payload_key_rotation(
+        &mut harness,
+        account1.clone(),
+        *account1.address(),
+        10,
+        account2.privkey.clone(),
+        account2.pubkey.clone(),
+    );
+    // rotate account1's keypair to account2
+    account1.rotate_key(account2.privkey, account2.pubkey);
```

Comment:
@jjleng do we plan to support updating the original account's private key and public key after key rotation in cmd and SDK?

Reply:
for CLI, users can save the new pri_k and pub_k into a new profile. We don't override the existing profile.